CHANGES.rst

```diff
@@ -1,5 +1,6 @@
 2.0.0 (Unreleased)
 ==================
+- [BREAKING] Removed make_result method from View and Query classes.
```

Comment:
I suppose if we're going to 100% accurate about what to do now for a developer to "make a result" then I think this should read something like this:

```
[BREAKING] Removed the make_result method from View and Query classes.  If you need to make a query or view result now, use ``CloudantDatabase.get_query_result``, ``CouchDatabase.get_view_result``, or the ``View.custom_result`` context manager, or you can use the ``Result`` and ``QueryResult`` classes directly and construct a result object that way.
```